scenarios/httpclient.benchmarks.yml

```diff
@@ -0,0 +1,189 @@
+﻿# Local run:
+# crank --config "C:\Users\knatalia\dev\git\Benchmarks\scenarios\httpclient.benchmarks.yml" --scenario httpclient-kestrel-get --profile local
+
+imports:
+  - https://github.com/aspnet/Benchmarks/blob/main/scenarios/aspnet.profiles.yml?raw=true
+
+variables:
+    serverPort: 5000
+    httpVersion: "1.1"
+    useHttps: true
+    numberOfHttpClients: 1
+    concurrencyPerHttpClient: 200
```

Comment:
While experimenting, I've found out that having anything other than 1 as a default is actually not convenient. E.g. `concurrencyPerHttpClient` doesn't make sense with WRK, so it will be plainly ignored, but while comparing WRK with HttpClient you want to be sure that you are benchmarking the same thing. The same is with comparing "1 client x N threads" vs "N clients x 1 thread", easier to override 1 variable instead of 2.